jsk_pcl_ros/src/color_filter_nodelet.cpp

```diff
@@ -241,16 +241,19 @@ namespace jsk_pcl_ros
   {
     ConnectionBasedNodelet::onInit();
 
-    filter_instance_ = pcl::ConditionalRemoval<pcl::PointXYZRGB>(true);
     updateCondition();
+    bool keep_organized_;
```

Comment:
Please remove `_` suffix. 
Probably it is the naming rule for class attribute. https://google.github.io/styleguide/cppguide.html#Variable_Names